### PR TITLE
[zh-cn] Update Element API index

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Finish translation

### CODE REVIEW

File: files/zh-cn/web/api/element/index.md [hunk 1]
```diff
@@ -1,344 +1,455 @@
...
+- {{domxref("Element.ariaRequired")}}
+  - : [`aria-required`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-required) 属性的字符串，表示在提交表单前需要用户输入。
+- {{domxref("Element.ariaRoleDescription")}}
+  - : 反映 [`aria-roledescription`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription) 属性的字符串，它定义了一个元素的角色的人类可读、作者本地化描述。
+- {{domxref("Element.ariaRowCount")}}
+  - : 反映 [`aria-rowcount`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-rowcount) 属性的字符串，该属性定义了表格、网格或树状网格中的总行数。
+- {{domxref("Element.ariaRowIndex")}}
+  - : 反映 [`aria-rowindex`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-rowindex) 属性的字符串，它定义了元素的行索引或相对于表格、网格或树状网格中总行数的位置。
+- {{domxref("Element.ariaRowIndexText")}} {{experimental_inline}}
+  - : 反映 [`aria-rowindextext`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-rowindext) 属性的字符串，它定义了 aria-rowindex 的人类可读文本替代。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/index.md:156]:
```suggestion
  - : 反映 [`aria-rowindextext`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-rowindextext) 属性的字符串，它定义了 aria-rowindex 的人类可读文本替代。
```

File: files/zh-cn/web/api/element/index.md [hunk 2]
```diff
@@ -1,344 +1,455 @@
...
+- {{domxref("Element.ariaRowSpan")}}
+  - : 反映 [`aria-rowspan`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-rowspan) 属性的字符串，该属性定义了表格、网格或树型网格中单元格或网格单元格所跨行数。
+- {{domxref("Element.ariaSelected")}}
+  - : 反映 [`aria-selected`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-selected)属性的字符串，表示具有选定状态的元素的当前“选定”状态。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/index.md:160]:
```suggestion
  - : 反映 [`aria-selected`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-selected) 属性的字符串，表示具有选定状态的元素的当前“选定”状态。
```

File: files/zh-cn/web/api/element/index.md [hunk 3]
```diff
@@ -1,344 +1,455 @@
...
+- {{domxref("Element.ariaSetSize")}}
+  - : 反映 [`aria-setize`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-setize) 属性的字符串，该属性定义了当前列表项或树状项集合中的项数。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/index.md:162]:
```suggestion
  - : 反映 [`aria-setsize`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-setsize) 属性的字符串，该属性定义了当前列表项或树状项集合中的项数。
```

File: files/zh-cn/web/api/element/index.md [hunk 4]
```diff
@@ -1,344 +1,455 @@
...
+- {{domxref("Element.ariaValueMin")}}
+  - : 反映 [`aria-valueMin`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-valuemin) 属性的字符串，该属性定义了范围部件的最小允许值。
+- {{domxref("Element.ariaValueNow")}}
+  - : 反映 [`aria-valueNow`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-valuenow) 属性的字符串，该属性定义了范围部件的当前值。
+- {{domxref("Element.ariaValueText")}}
+  - : 反映 [`aria-valuetext`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-valuetext) 属性的字符串，该属性为范围部件定义了 aria-valuenow 的人类可读文本替代值。
+
+## 实例方法
+
+_`Element` 继承其父级 {{DOMxRef("Node")}} 和父级 {{DOMxRef("EventTarget")}} 的方法。_
```
Comment (yin1999) [inline files/zh-cn/web/api/element/index.md:176]: How about this:

```suggestion
_`Element` 继承 {{DOMxRef("Node")}} 及其父接口 {{DOMxRef("EventTarget")}} 的方法。_
```

File: files/zh-cn/web/api/element/index.md [hunk 5]
```diff
@@ -1,344 +1,455 @@
...
-  - : Returns a {{DOMxRef("StylePropertyMapReadOnly")}} interface which provides a read-only representation of a CSS declaration block that is an alternative to {{DOMxRef("CSSStyleDeclaration")}}.
-- {{DOMxRef("EventTarget.dispatchEvent()")}}
-  - : Dispatches an event to this node in the DOM and returns a {{jsxref("Boolean")}} that indicates whether no handler canceled the event.
-- {{DOMxRef("Element.getAnimations()")}} {{Experimental_Inline}}
-  - : Returns an array of Animation objects currently active on the element.
+  - : 为指定元素附加影子 DOM 树，并返回指向其 {{DOMxRef("ShadowRoot")}} 的引用。
+- {{DOMxRef("Element.before()")}}
+  - : 在 `Element` 父节点的子节点列表中插入一组 {{domxref("Node")}} 对象或字符串，位于 `Element` 之前。
+- {{DOMxRef("Element.closest()")}}
+  - : 返回 {{DOMxRef("Element")}} 当前元素（或当前元素本身）最接近的祖先，且与参数中给定的选择器匹配。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/index.md:189]:
```suggestion
  - : 返回 `Element` 当前元素（或当前元素本身）最接近的祖先，且与参数中给定的选择器匹配。
```

File: files/zh-cn/web/api/element/index.md [hunk 6]
```diff
@@ -1,344 +1,455 @@
...
+- {{DOMxRef("Element.computedStyleMap()")}}
+  - : 返回一个 {{DOMxRef("StylePropertyMapReadOnly")}} 接口，该接口提供 CSS 声明块的只读表示，可替代 {{DOMxRef("CSSStyleDeclaration")}} 。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/index.md:191]:
```suggestion
  - : 返回一个 {{DOMxRef("StylePropertyMapReadOnly")}} 接口，该接口提供 CSS 声明块的只读表示，可替代 {{DOMxRef("CSSStyleDeclaration")}}。
```

File: files/zh-cn/web/api/element/index.md [hunk 7]
```diff
@@ -1,344 +1,455 @@
...
 - {{DOMxRef("Element.getAttribute()")}}
-  - : Retrieves the value of the named attribute from the current node and returns it as an {{jsxref("Object")}}.
+  - : 从当前节点读取指定属性的值，并以字符串形式返回。
 - {{DOMxRef("Element.getAttributeNames()")}}
-  - : Returns an array of attribute names from the current element.
+  - : 返回当前元素的属性名称数组。
+- {{DOMxRef("Element.getAttributeNode()")}}
+  - : 从当前节点获取指定属性的节点表示，并以 {{DOMxRef("Attr")}} 的形式返回。
+- {{DOMxRef("Element.getAttributeNodeNS()")}}
+  - : 从当前节点读取指定名称和命名空间的属性的节点表示，并以 {{DOMxRef("Attr")}}} 的形式返回。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/index.md:201]:
```suggestion
  - : 从当前节点读取指定名称和命名空间的属性的节点表示，并以 {{DOMxRef("Attr")}} 的形式返回。
```

File: files/zh-cn/web/api/element/index.md [hunk 8]
```diff
@@ -1,344 +1,455 @@
...
 
 ## 事件
 
-Listen to these events using `addEventListener()` or by assigning an event listener to the `oneventname` property of this interface.
+使用 `addEventListener()` 或将事件监听器分配给此接口的 `oneventname` 属性来监听这些事件。
+
+- {{domxref("Element/afterscriptexecute_event","afterscriptexecute")}} {{Non-standard_Inline}}
+  - : 执行脚本时触发此事件。
+- {{domxref("Element/beforematch_event", "beforematch")}} {{Experimental_Inline}}
+  - : 触发处于 [_hidden until found_](/zh-CN/docs/Web/HTML/Global_attributes/hidden) 状态的元素，此时浏览器即将显示其内容，因为用户已通过“在页面中查找”功能或片段导航找到了该内容。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/index.md:290]: Another suggestion is "被找到前隐藏", we may need to investigate further

```suggestion
  - : 浏览器在（处于[_隐藏但允许查找_](/zh-CN/docs/Web/HTML/Global_attributes/hidden)状态的）元素因用户已通过“在页面中查找”功能或片段导航找到了该内容而即将显示时触发。
```

File: files/zh-cn/web/api/element/index.md [hunk 9]
```diff
@@ -1,344 +1,455 @@
...
+l10n:
+  sourceCommit: e83aa62ce6bf2e2bc44f55a3fb73b63d724fa6a6
 ---
 
 {{APIRef("DOM")}}
 
-**`Element`** 是一个通用性非常强的基类，所有 {{DOMxRef("Document")}} 对象下的对象都继承自它。这个接口描述了所有相同种类的元素所普遍具有的方法和属性。一些接口继承自 `Element` 并且增加了一些额外功能的接口描述了具体的行为。例如， {{DOMxRef("HTMLElement")}} 接口是所有 HTML 元素的基本接口，而 {{DOMxRef("SVGElement")}} 接口是所有 SVG 元素的基础。大多数功能是在这个类的更深层级（hierarchy）的接口中被进一步制定的。
+**`Element`** 是一个通用性非常强的基类，所有 {{DOMxRef("Document")}} 对象下的对象都继承自它。这个接口描述了所有相同种类的元素所普遍具有的方法和属性。一些接口继承自 `Element` 并且增加了一些额外功能的接口描述了具体的行为。
+
+例如，{{DOMxRef("HTMLElement")}} 接口是所有 HTML 元素的基本接口，而 {{DOMxRef("SVGElement")}} 接口是所有 SVG 元素的基础。大多数功能是在这个类的更深层级的接口中被进一步制定的。
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/web/api/element/index.md:12]:
```suggestion
例如，{{DOMxRef("HTMLElement")}} 接口是所有 HTML 元素的基本接口。同样，{{DOMxRef("SVGElement")}} 接口是所有 SVG 元素的基本接口，而 {{DOMxRef("MathMLElement")}} 接口则是 MathML 元素的基础接口。大多数功能是在这个类的更深层级的接口中被进一步制定的。
```

File: files/zh-cn/web/api/element/index.md [hunk 10]
```diff
@@ -1,344 +1,455 @@
...
 title: Element
 slug: Web/API/Element
+l10n:
+  sourceCommit: e83aa62ce6bf2e2bc44f55a3fb73b63d724fa6a6
 ---
 
 {{APIRef("DOM")}}
 
-**`Element`** 是一个通用性非常强的基类，所有 {{DOMxRef("Document")}} 对象下的对象都继承自它。这个接口描述了所有相同种类的元素所普遍具有的方法和属性。一些接口继承自 `Element` 并且增加了一些额外功能的接口描述了具体的行为。例如， {{DOMxRef("HTMLElement")}} 接口是所有 HTML 元素的基本接口，而 {{DOMxRef("SVGElement")}} 接口是所有 SVG 元素的基础。大多数功能是在这个类的更深层级（hierarchy）的接口中被进一步制定的。
+**`Element`** 是一个通用性非常强的基类，所有 {{DOMxRef("Document")}} 对象下的对象都继承自它。这个接口描述了所有相同种类的元素所普遍具有的方法和属性。一些接口继承自 `Element` 并且增加了一些额外功能的接口描述了具体的行为。
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/web/api/element/index.md:10]: **`Element`** 是最通用的基类，{{DOMxRef("Document")}} 中的所有元素对象（即表示元素的对象）都继承自它。它只具有各种元素共有的方法和属性。更具体的类则继承自 `Element`。

[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/web/api/element/index.md:10]:
```suggestion
**`Element`** 是最通用的基类，{{DOMxRef("Document")}} 中的所有元素对象（即表示元素的对象）都继承自它。它只具有各种元素共有的方法和属性。更具体的类则继承自 `Element`。
```

File: files/zh-cn/web/api/element/index.md (hunk 8 above)
Comment (jasonren0403) [reply files/zh-cn/web/api/element/index.md:290]: "被找到前隐藏" seems better? I have no idea between these two.

Comment (yin1999) [reply files/zh-cn/web/api/element/index.md:290]: > "被找到前隐藏" seems better?

Ok, let's use this one for now.

Comment (jasonren0403) [reply files/zh-cn/web/api/element/index.md:290]:
```suggestion
  - : 浏览器在（处于[_被找到前隐藏_](/zh-CN/docs/Web/HTML/Global_attributes/hidden)状态的）元素因用户已通过“在页面中查找”功能或片段导航找到了该内容而即将显示时触发。
```

**Review by yin1999 — APPROVED** [review]
Thank you @jasonren0403

**Pull request merged**